src/fides/api/ops/service/connectors/saas/connector_registry_service.py

```diff
@@ -189,10 +202,9 @@ def update_saas_configs(registry: ConnectorRegistry, db: Session) -> None:
                         saas_config_instance,
                     )
                 except Exception:
-                    logger.error(
+                    logger.exception(
```

Comment:
i would say you're fine to ignore this codecov miss since it already was a miss and you just updated the log statement